<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AMySettingsForm</class>
 <widget class="QWidget" name="AMySettingsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>358</width>
    <height>212</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Настройки</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="permanentFoodGenerationCheckBox">
       <property name="text">
        <string>Постоянная генерация еды</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="foodGenerationChanceEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Шанс генерации еды</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="realTimeReproductionCheckBox">
     <property name="text">
      <string>Размножение в реальном времени</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="openWorldRadioButton">
        <property name="text">
         <string>Открытый мир</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="closeWordRadioButton">
        <property name="text">
         <string>Закрытый мир</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Размер программы живых объектов</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="liveObjectProgrammSizeEdit">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="liveObjectChangeableProgrammSizeCheckBox">
     <property name="text">
      <string>Размер программы может увеличиватся</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="liveObjectsIfLengthCheckBox">
     <property name="text">
      <string>Живые обьекты видят на расстоянии</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="applyButton">
     <property name="text">
      <string>Применить</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
